src/components/Map.js

```diff
@@ -0,0 +1,217 @@
+import React, { useEffect, useState } from 'react'
+
+// Start Openlayers imports
+import { 
+    Map,
+    View
+ } from 'ol'
+import {
+    Tile as TileLayer,
+} from 'ol/layer'
+import {
+    XYZ as XYZSource,
+    TileWMS as TileWMSSource
+} from 'ol/source'
+import { 
+    ScaleLine,
+    ZoomSlider,
+    MousePosition,
+    OverviewMap,
+    defaults as DefaultControls
+} from 'ol/control'
+
+// End Openlayers imports
+
+const OLMap = (props) => {
+  const [height, setHeight] = useState(400)
+  
+  const updateDimensions = () => {
+    const h = window.innerWidth >= 992 ? window.innerHeight : 400
+    setHeight({height: h})
```

Comment:
this should just be `setHeight(h)`.  the `{height: h}` looks like leftover from old setState